xml/System/Span`1.xml

```diff
@@ -760,10 +852,18 @@
         <Parameter Name="destination" Type="System.Span&lt;T&gt;" />
       </Parameters>
       <Docs>
-        <param name="destination">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="destination">The target of the copy operation.</param>
+        <summary>Attempts to copy the current <see cref="T:System.Span`1" /> to a destination <see cref="T:System.Span`1" /> and returns a value that indicates whether the copy operation succeeded.</summary>
+        <returns><see langword="true" /> if the copy operation succeeded; otherwise, <see langword="false" />.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+
+If the source and `destination` overlap, this method places the original values in a temporary location before `destination` is overwritten.
```

Comment:
The API doesn't actually place the original values in a temporary location. It only behaves like this, semantically. The way it is currently written, it conveys the behavior imprecisely.

> this method behaves **as if** the original values are in a temporary location before the destination is overwritten.